packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -298,7 +298,8 @@ export const sandboxChanged = withLoadApp<{
   }
 
   try {
-    const sandbox = await effects.api.getSandbox(newId);
+    const params = state.activeTeam ? { teamId: state.activeTeam } : undefined;
```

Comment:
nitpick: I would use null to suggest the absence of the value, but if that calls for an extra conditional further down the line to check if params is null or not, undefined is also good